swatch-core/src/main/resources/swatch-core/application.yaml

```diff
@@ -8,8 +8,6 @@
 # result in an exception at start-up when the clowder property cannot be successfully resolved.
 # The values can still be overridden using environment variables as they have a higher precedence
 # than the defaults defined here.
-KAFKA_BOOTSTRAP_HOST: ${clowder.kafka.brokers[0].hostname:localhost}
-KAFKA_BOOTSTRAP_PORT: ${clowder.kafka.brokers[0].port:9092}
 KAFKA_USERNAME: ${clowder.kafka.brokers[0].sasl.username:}
 KAFKA_PASSWORD: ${clowder.kafka.brokers[0].sasl.password:}
```

Comment:
Do we need to worry about these properties have multiple values too?  I don't have the clowder JSON in front of me, but just want to confirm we won't have a problem trying to connect to `broker[1]` with the username for `broker[0]`

Reply:
Very fair point indeed! In my first version, I also ported the SASL properties, but then I thought I went too far. 
But if you're ok, I'm updating the PR including these changes as well.